README.md

```diff
@@ -57,8 +57,10 @@ optional arguments:
   --debug               enables debug level logging if set
 ```
 
+Example
+
 ```
-parse-account-statements.py  --type mintos src/test/testdata/mintos.csv
+./parse-account-statements.py  --type mintos src/test/testdata/mintos.csv
```

Comment:
Under which OS are you running the script? Maybe the section should get an appropriate heading. e.g. On Linux: ...